packages/toolpad-components/src/DataGrid.tsx

```diff
@@ -359,7 +359,10 @@ const DataGridComponent = React.forwardRef(function DataGridComponent(
   );
 
   return (
-    <div ref={ref} style={{ height: heightProp, minHeight: '100%', width: '100%' }}>
+    <div
+      ref={ref}
+      style={{ height: heightProp, minHeight: '100%', width: '100%', colorScheme: 'auto' }}
+    >
```

Comment:
We are not solving https://github.com/mui/mui-toolpad/pull/988#discussion_r981281610

For example https://toolpad-pr-988.onrender.com/_toolpad/app/cl6xm8w6n0616arpkpgmr5219/pages/cl6xm8w9u0001arpkdgfndg7n

<img width="616" alt="Screenshot 2022-10-06 at 00 32 34" src="https://user-images.githubusercontent.com/3165635/194175912-d847ea72-277b-4212-8ea6-78521bb29eee.png">

```suggestion
    <div ref={ref} style={{ height: heightProp, minHeight: '100%', width: '100%' }}>
```

Plus, 'auto' is not an accepted value: https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme (Chrome might not yet autocomplete the CSS property correctly). It's like you had set the initial value, so `normal`: https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme#formal_definition.

Reply:
For solving the root cause, I think that we miss the use of `<CssBaseline enableColorScheme />` for the theme inside the runtime toolpad app.

Reply:
> For solving the root cause, I think that we miss the use of <CssBaseline enableColorScheme /> for the theme inside the runtime toolpad app.

Opening https://github.com/mui/mui-toolpad/pull/1100 for that